RoaringBitmap/src/test/java/org/roaringbitmap/buffer/TestRoaringBitmap.java

```diff
@@ -3965,4 +3965,19 @@ public void issue623() {
     }
   }
 
+
+
+    @Test
+    public void test1235() {
+        MutableRoaringBitmap r = MutableRoaringBitmap.bitmapOf(1, 2, 3, 5);
+        r.flip(4);
+        assertEquals(r,  MutableRoaringBitmap.bitmapOf(1, 2, 3, 4, 5));
```

Comment:
@xtonik If there are bugs, please demonstrate them with code such as this. We only care about bugs affecting our users. If there is code that does not do what you expect, it is not a bug *unless* you can demonstrate a user-visible issue.